app/views/products/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Editing Product</h1>
```

Comment:
@mattre3 
Reinicjalizuj aplikacje jako zwykle api bez front endu, zrobisz to za pomocą rails new $app -d=postgresql --api, gdzie $app to nazwa twojej aplikacji.

Reply:
Muszę zainicjalizować nową apke, czy da się to jakoś zrobić w katalogu obecnej?